File: 0191-number-of-1-bits/solution.cpp
class Solution {
public:
    int hammingWeight(uint32_t n) 
    {
        int numOnes = 0;
        while(n)
        {
            (n&1)?numOnes++:(0);
            n>>=1;
        }
        return numOnes;
    }
};

File: 0191-number-of-1-bits/solution.py
class Solution(object):
    def hammingWeight(self, n):
        """
        :type n: int
        :rtype: int
        """
        numOnes = 0
        while n:
            if n&1 == 1:
                numOnes +=1
            n = n >>1
        return numOnes
